### ISSUE TITLE
Self contained Windows/Linux/MacOS Builds?

### ISSUE THREAD

**Issue by harrypm** [sandflow/ttconv#419]
Hello, I am just wondering if self contained binarys could be made for archival/offline usage?

Currently wanting to impliment SCC --> SRT conversion in the tbc-video-export tool for [ld/tape/cvbs-decode](https://github.com/oyvindln/vhs-decode/wiki/) projects and our metadata export tool creates standard SCC data and for player/device reading felexibility this needs to be in srt.

But sadly this requires a self contained exacutable to be viable for use in the windows version outherwise this forces the need of installing external dependencys or using WSL2 witch is a lot harder to make into self contained toolkit packages for media archives.

[ ] - Windows
[ ] - MacOS
[ ] - MacOS Arm
[ ] - Linux (app image)

**Comment by palemieux** [comment]
Do you mean something like [Pyinstaller](https://pyinstaller.org/en/stable/)?

**Comment by harrypm** [comment]
> Do you mean something like [Pyinstaller](https://pyinstaller.org/en/stable/)?

Yeah that could work, not sure what if anything might have to be tweaked for it though.
